### PR TITLE
MOTECH-2256 - Atom Client Module Documentation

### PR DESCRIPTION
**Pull request by frankhuster**
Atom Client module documentation

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -12,6 +12,11 @@ Collects alerts for users in an inbox-like container
 
 Provides appointment scheduling and reminders
 
+:std:ref:`Atom Client <atom-client-module>`
+---------------------------------------------
+
+Fetches Atom feed(s) and publishes events when content changes
+
```
Comment (llavoie) [inline docs/source/index.rst:19]: I think this change actually needs to be made in the motech repo instead. There's a file at docs/source/modules/index.rst, which is actually the one rendered here: http://docs.motechproject.org/en/latest/modules/index.html

File: docs/source/atomclient.rst [hunk 2]
```diff
@@ -0,0 +1,104 @@
+.. _atom-client-module:
+
+==================
+Atom Client Module
+==================
+
+Description
+===========
+
+This module will periodically fetch from one or more `Atom feed <https://en.wikipedia.org/wiki/Atom_%28standard%29>`_ sources and send a MOTECH event for each feed content that changed since the last fetch.
```
Comment (llavoie) [inline docs/source/atomclient.rst]: "each feed content" sounds a bit awkward. Should it be each "each feed item" or something like that?

File: docs/source/atomclient.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+=====
+
+When fetching and for each feed content item that changed since the last time a fetch occurred, the Atom Client module will produce a ``Feed Change`` task trigger with the following payload data:
+
+
+    ``url``
+        The URL identifying the Atom feed
+
+    ``published_date``
+        When the content item ws created
```
Comment (llavoie) [inline docs/source/atomclient.rst]: typo: "was", not "ws"

File: docs/source/atomclient.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+    ``updated_date``
+        When the content item was changed last
+
```
Comment (llavoie) [inline docs/source/atomclient.rst:69]: nitpick: "last changed" is better than "changed last"

File: docs/source/index.rst (hunk 1 above)
Comment (frankhuster) [inline docs/source/index.rst:19]: As discussed, @llavoie is already working on a documentation PR in the motech/motech repo and will add that.

**Comment by llavoie** [comment]
@frankhuster LGTM, feel free to merge once the build passes

**Comment by pgesek** [comment]
test this please

**Pull request merged**